report_thesis/src/sections/introduction.tex

```diff
@@ -24,7 +24,8 @@ \section{Introduction}\label{sec:introduction}
 These models were identified and selected through a combination of extensive literature review and the consideration of unconventional approaches not typically covered in the \gls{libs}-based calibration literature.
 The ten models fall into three categories: Ensemble learning models, linear and regularization models, and neural network models.
 In addition to model exploration, we also investigate a selection of preprocessing techniques: scaling (including normalization), dimensionality reduction, and data transformation.
-Specifically, we designed and implemented a framework for experimental analysis using the automated hyperparameter optimization framework Optuna~\cite{optuna} to determine the most effective combinations of preprocessing methods and models for each regression target.
+Specifically, we designed and implemented a framework for experimental analysis using the automated hyperparameter optimization framework Optuna~\cite{optuna_2019}.
```

Comment:
I think the Optuna part is best mentioned in the Method section, where we describe the whole thing. Especially since MLFlow plays a very large role, as well.

```suggestion
Specifically, we designed and implemented a framework for large-scale experimental analysis of models and data preprocessing steps.
```

Reply:
It was specifically requested by the supervisors that we name drop which framework in a previous feedback round.